model/job.go

```diff
@@ -88,6 +90,8 @@ type JobUpdate struct {
 
 	CronSchedule *string    `json:"cron_schedule,omitempty"`
 	ExecuteAt    *time.Time `json:"execute_at,omitempty"`
+
+	Tags *[]string `json:"tags,omitempty"`
```

Comment:
Don't need an pointer to the array for nil checks, because arrays can also have nil values.

Reply:
A pointer is required for differentiating between removing all tags (empty slice) / not updating tags (nil pointer).